components/cam/src/chemistry/utils/modal_aero_wateruptake.F90

```diff
@@ -15,18 +15,17 @@ module modal_aero_wateruptake
 use cam_logfile,      only: iulog
 use ref_pres,         only: top_lev => clim_modal_aero_top_lev
 use phys_control,     only: phys_getopts
-use cam_abortutils,   only: endrun
-use crmdims,          only: crm_nx, crm_ny, crm_nz, crm_nx_rad, crm_ny_rad
+use cam_abortutils,       only: endrun
+
 implicit none
 private
 save
 
 public :: &
-  modal_aero_wateruptake_reg,   &
-  modal_aero_wateruptake_init,  &
-  modal_aero_wateruptake_dr,    &
-  modal_aero_kohler
-  
+   modal_aero_wateruptake_init, &
+   modal_aero_wateruptake_dr
+
+public :: modal_aero_wateruptake_reg
```

Comment:
Is there a reason to declare modal_aero_wateruptake_reg separately? instead of including it within line 24-26?

Reply:
Never mind, it must be how it was done in E3SM